uktrade/data-hub-frontend
# Handle submission of the new interaction form

## Description of change

Handle the submit callback of the new interaction form.

## Test instructions

1. Go to [the new interaction form](https://data-hub-fro-new-intera-nsuub4.herokuapp.com/interactions/create?company_id=5bb239e4-b227-4a84-bc2e-3420fc129447)
2. Confirm if the new form works the same as [the old interaction form](https://data-hub-fro-new-intera-nsuub4.herokuapp.com/companies/5bb239e4-b227-4a84-bc2e-3420fc129447/interactions/create)
